tests/components/devolo_home_network/snapshots/test_init.ambr

```diff
@@ -5,6 +5,10 @@
     'config_entries': <ANY>,
     'configuration_url': 'http://192.0.2.1',
     'connections': set({
+      tuple(
+        'mac',
+        'aa:bb:cc:dd:ee:ff',
+      ),
```

Comment:
Why not simply: `{('mac', 'aa:bb:cc:dd:ee:ff')}`

Reply:
I'm not choosing the format of the snapshot. [Syrupy](https://developers.home-assistant.io/docs/development_testing#snapshot-testing) does that.

Reply:
It is generated by Syrupy, yes